test/integration/query-interface.test.js

```diff
@@ -63,6 +63,45 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
           expect(tableNames).to.deep.equal(['my_test_table']);
         });
     });
+
+    if (dialect !== 'sqlite' && dialect !== 'postgres') {
+      // NOTE: sqlite doesn't allow querying between databases and
+      // postgres requires creating a new connection to create a new table.
+      it('should not show tables in other databases', function() {
+        return this.queryInterface
+          .createTable('my_test_table1', { name: DataTypes.STRING })
+          .then(() => this.sequelize.query('CREATE DATABASE my_test_db'))
+          .then(() => this.sequelize.query(`CREATE TABLE my_test_db${dialect === 'mssql' ? '.dbo' : ''}.my_test_table2 (id INT)`))
```

Comment:
having a test `dialect === 'mssql'`  inside a test `if (dialect !== 'sqlite' && dialect !== 'postgres') {` ?

Reply:
because `mssql` requires listing the schema (`dbo`) when creating the table but `mysql` and `mariadb` don't.